upgrade_test.py

```diff
@@ -392,6 +392,17 @@ def test_upgrade_cql_queries(self):
         self.verify_db_data()
         self.verify_stress_thread(stress_queue)
 
+    def fill_and_verify_db_data(self, note, pre_fill=False, post_fill=True):
```

Comment:
maybe change `post_fill` to `rewrite_data`, to make it a bit clearer ?

Reply:
Fixed.